### PR TITLE
Added queue to worker

### PR DESCRIPTION
**Pull request by cccs-cat001**
**Description of the change**

I added the possibility to spawn workers with a queue. 

**Benefits**

This allows workers to be directed to a specific workload, for instance different workers could have different libraries installed, having a queue specified will help direct tasks to the proper workers.

**Possible drawbacks**

If a user ends up using just this image and doesn't define the `AIRFLOW_QUEUE` environment variable, it will not launch. I use it in conjunction with the bitnami/airflow helm chart, and I've modified the chart there to specify a queue for the workers. see [here](https://github.com/bitnami/charts/pull/1703)

**Applicable issues**

**Additional information**

### CODE REVIEW

File: 1/debian-9/rootfs/run.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@
 USER=airflow
 DAEMON=airflow
 EXEC=$(which $DAEMON)
-START_COMMAND="${EXEC} worker | tee /opt/bitnami/airflow/logs/airflow-worker.log"
+START_COMMAND="${EXEC} worker -q $AIRFLOW_QUEUE | tee /opt/bitnami/airflow/logs/airflow-worker.log"
```
Comment (javsalgar) [inline 1/debian-9/rootfs/run.sh]: Would it make sense to add and `if` statement and set START_COMMAND accordingly?

Comment (cccs-cat001) [reply 1/debian-9/rootfs/run.sh]: It probably would :)

File: 1/debian-9/rootfs/run.sh [hunk 2]
```diff
@@ -6,8 +6,11 @@
 USER=airflow
 DAEMON=airflow
 EXEC=$(which $DAEMON)
-START_COMMAND="${EXEC} worker | tee /opt/bitnami/airflow/logs/airflow-worker.log"
-
+if [ $AIRFLOW_QUEUE ]; then
+    START_COMMAND="${EXEC} worker -q $AIRFLOW_QUEUE | tee /opt/bitnami/airflow/logs/airflow-worker.log"
+else
+    START_COMMAND="${EXEC} worker | tee /opt/bitnami/airflow/logs/airflow-worker.log"
+fi
```
Comment (sameersbn) [inline 1/debian-9/rootfs/run.sh]: You could do something like this instead

```bash
START_COMMAND="${EXEC} ${AIRFLOW_QUEUE:+-q $AIRFLOW_QUEUE} worker | tee /opt/bitnami/airflow/logs/airflow-worker.log"
```

But I wonder is we add a `AIRFLOW_EXTRA_ARGS` option instead. wdyt @javsalgar ?

@cccs-cat001 You should also updated the README to reflect this change

**Comment by sameersbn** [comment]
Thanks for the update. Would like to know @javsalgar thoughts on adding a `AIRFLOW_EXTRA_ARGS`

File: README.md [hunk 3]
```diff
@@ -133,6 +133,7 @@ If you want to run the application manually instead of using `docker-compose`, t
     -e AIRFLOW_DATABASE_NAME=bitnami_airflow \
     -e AIRFLOW_DATABASE_USERNAME=bn_airflow \
     -e AIRFLOW_DATABASE_PASSWORD=bitnami1 \
+    -e AIRFLOW_QUEUE=new_queue \
```
Comment (sameersbn) [inline README.md]: you should document `AIRFLOW_QUEUE` in the [Configuration](https://github.com/bitnami/bitnami-docker-airflow-worker/blob/master/README.md#configuration) section of the README

**Comment by sameersbn** [comment]
Thanks for the contribution @cccs-cat001

**Pull request merged**